package.json

```diff
@@ -2,10 +2,14 @@
   "name": "electrum-proxy-middleware",
   "version": "1.0.0",
   "description": "",
-  "main": "src/index.js",
+  "main": "dist/src/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "docs": "swagger-jsdoc -d swaggerfile.js"
+    "docs": "swagger-jsdoc -d swaggerfile.js",
+    "build-bitcore-lib": "browserify -r bitcore-lib -s bitcore-lib -o src/assets/libs/bitcore-lib.js",
+    "build-zcash-bitcore-lib": "browserify -r zcash-bitcore-lib -s zcash-bitcore-lib -o src/assets/libs/zcash-bitcore-lib.js",
```

Comment:
no sence to browserify "bitcore-lib" & "zcash-bitcore-lib" they was created for nodejs

Reply:
fixed